DataFormats/Phase2ITPixelCluster/interface/Phase2ITPixelCluster.h

```diff
@@ -0,0 +1,279 @@
+#ifndef DataFormats_Phase2ITPixelCluster_Phase2ITPixelCluster_h
+#define DataFormats_Phase2ITPixelCluster_Phase2ITPixelCluster_h
+
+//---------------------------------------------------------------------------
+//!  \class Phase2ITPixelCluster
+//!  \brief Pixel cluster -- collection of neighboring pixels above threshold
+//!
+//!  Class to contain and store all the topological information of pixel clusters:
+//!  charge, global size, size and the barycenter in x and y
+//!  local directions. It builds a vector of SiPixel (which is
+//!  an inner class) and a container of channels. 
+//!
+//!  March 2007: Edge methods moved to RectangularPixelTopology class (V.Chiochia)
+//!  Feb 2008: Modify the Pixel class from float to shorts
+//!  May   2008: Offset based packing (D.Fehling / A. Rizzi)
+//!  Sep 2012: added Max back, removed detId (V.I.)
+//!  sizeX and sizeY now clipped at 127
```

Comment:
These history comments are copied from the original file and are irrelevant and confusing here. Please delete them.